src/components/question-wrapper.tsx

```diff
@@ -14,9 +14,15 @@ type TabName = "Correct" | "Distractors" | "TeacherTip" | "Exemplar";
 
 const LARA_MULTIPLE_CHOICE = "Embeddable::MultipleChoice";
 const LARA_INTERACTIVES = [ "MwInteractive", "ImageInteractive", "VideoInteractive" ];
+const MANAGED_INTERACTIVE = "ManagedInteractive";
 
 export const isMCQuestion = (wrappedEmbeddableContext: any) => {
-  return wrappedEmbeddableContext.type === LARA_MULTIPLE_CHOICE;
+  const authState = wrappedEmbeddableContext.authored_state
+                    ? JSON.parse(wrappedEmbeddableContext.authored_state)
+                    : undefined;
+  return wrappedEmbeddableContext.type === LARA_MULTIPLE_CHOICE
+         || (wrappedEmbeddableContext.type === MANAGED_INTERACTIVE && authState
+             && authState.questionType === "multiple_choice");
```

Comment:
I know we talked about this in the context of the Activity Player, but does the plugin have access to the interactive's supportedFeatures so that it could avoid hard-coding the types of activities it works with?

Reply:
I think we need to add extra support for that.